tools/add_font.py

```diff
@@ -23,6 +55,30 @@
                      ' for a -ext subset.')
 
 
+def family_url(name):
+  url_prefix = 'http://fonts.google.com/specimen/'
+  dl_name = name.replace(' ', '+')
+  return url_prefix + dl_name
+
+
+def family_exists(name):
+  """Check if a font family exists on fonts.google.com"""
+  url = family_url(name)
+  request = requests.get(url)
+  if request.status_code == 200:
+      return True
+  return False
+
+
+def _GetFamilyMetaData(name):
+  """Get family information from the frontend GF api"""
+  api_prefix = 'http://fonts.google.com/metadata/fonts/'
```

Comment:
Anything public *should* exist in this repository. We should build out this tool based on that assumption and fix it if it's violated. If the tool can avoid network hits, such as by retaining the values from the pre-existing file that makes it much easier to test and maintain so I would prefer to avoid ever updating it to have network dependencies.

@davelab6 Are you aware of this drift? Should we embark on an effort to bring repo & fonts.google.com back into sync?